cromwell-helm/templates/postgres.yaml

```diff
@@ -0,0 +1,52 @@
+###########################
+# Deployment
+###########################
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "app.fullname" . }}-postgres-depl
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      component: postgres
```

Comment:
I don't know if I should make these labels more complicated (like we do for Cromwell, following Galaxy examples). TBH, I don't completely understand the point of having multiple labels.

Reply:
What are these labels used for? Do they show up somewhere or are they just for identification purposes?

Reply:
TBH, I'm still pretty confused about how labels are used. But here is the documentation-- https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/